src/schema.ts

```diff
@@ -2,10 +2,13 @@ import * as _ from "lodash"
 import * as mongoose from "mongoose"
 import { yamlConfig, levels } from "./config"
 import { NotFoundError } from "./error"
-import { accountPath } from "./ledger/ledger"
+import { accountPath } from "./ledger/accounts"
+import { Transaction } from "./ledger/schema"
 import { baseLogger } from "./logger"
 import { caseInsensitiveRegex } from "./utils"
 
+export { Transaction } from "./ledger/schema"
```

Comment:
what is the goal import/exporting this class?

Reply:
isn't this equivalent to `export Transaction`? because Transaction is already in the scope of this file with the import?

Reply:
fixed, changed to `export { Transaction }`.